launch/constants.py

```diff
@@ -7,7 +7,8 @@
 BATCH_TASK_PATH = "batch_job"
 BATCH_TASK_RESULTS_PATH = "batch_job"
 RESULT_PATH = "result"
-SCALE_LAUNCH_ENDPOINT = "https://api.scale.com/v1/hosted_inference"
-SCALE_LAUNCH_V1_ENDPOINT = "https://api.scale.com/v1/launch"
+DEFAULT_SCALE_ENDPOINT = "https://api.scale.com"
+HOSTED_INFERENCE_PATH = "/v1/hosted_inference"
```

Comment:
In the spirit of the previous names, maybe let's call this `SCALE_LAUNCH_V0_PATH` and `SCALE_LAUNCH_V1_PATH`.